docs/targets/s3.md

```diff
@@ -0,0 +1,67 @@
+# Event Target for AWS S3
+
+This event Target receives [CloudEvents][ce] over HTTP and invokes an AWS S3 endpoint.
+
+## Prerequisites
+
+1. AWS API key and secret
+
+1. ARN for the S3 bucket to store the event
+
+Consult the [Secrets](../guides/secrets.md) guide for more information about
+how to add the AWS API specific secrets.
+
+## Deploying an Instance of the Target
+
+Open the Bridge creation screen and add a Target of type `AWS S3`.
+
+![Adding an S3 Target](../images/aws-targets/aws-s3-bridge-create-1.png)
+
+In the Target creation form, provide a name for the event Target, and add the following information:
+
+* **AWS Secret**: Reference a [TriggerMesh secret][tm-secret] containing an AWS API key and Secret as discussed in the prerequisites
```

Comment:
[tm-secret] is not defined

Reply:
Good catch. It appears this was missing throughout the other docs, and will be updated here.

Reply:
This is taking me to a 404 page too 
<img width="1440" alt="Screen Shot 2020-11-12 at 11 57 32 AM" src="https://user-images.githubusercontent.com/54158919/98970743-48580600-24de-11eb-815d-f5c150120292.png">

Reply:
Thank you for catching this @daceynolan. Come to find out, the `[tm-secret]` reference requires a full URL instead of relying on a relative path for rendering. This has been corrected.

@sameersbn Since you did some of the sources work, you may need to address this as well.